### ISSUE TITLE
Is it possible to install with Pip? 

### ISSUE THREAD

**Issue by JemiloII** [bumble-tech/private-detector#9]
Is it possible to install with Pip? I use venv and don't want to use conda.

**Comment by Steeeephen** [comment]
Yeah definitely, the `environment.yaml` file shows the package versions needed - can just install them directly with pip

We recommended conda as it takes care of some of the issues around GPU drivers, but if you're on CPU/can take care yourself then it should be ok

**Comment by dkbarn** [comment]
I can't get this to work. The versions specified in the environment.yaml file are too old for tensorflow-gpu and tensorflow_addons:
```
$ venv/bin/pip install 'tensorflow-gpu==2.4.*'
ERROR: Could not find a version that satisfies the requirement tensorflow-gpu==2.4.* (from versions: 2.8.0rc0, 2.8.0rc1, 2.8.0, 2.8.1, 2.8.2, 2.8.3, 2.8.4, 2.9.0rc0, 2.9.0rc1, 2.9.0rc2, 2.9.0, 2.9.1, 2.9.2, 2.9.3, 2.10.0rc0, 2.10.0rc1, 2.10.0rc2, 2.10.0rc3, 2.10.0, 2.10.1, 2.11.0rc0, 2.11.0rc1, 2.11.0rc2, 2.11.0, 2.12.0)
ERROR: No matching distribution found for tensorflow-gpu==2.4.*

$ venv/bin/pip install 'tensorflow_addons==0.13.0'
ERROR: Could not find a version that satisfies the requirement tensorflow_addons==0.13.0 (from versions: 0.16.1, 0.17.0, 0.17.1, 0.18.0, 0.19.0, 0.20.0, 0.21.0, 0.22.0)
ERROR: No matching distribution found for tensorflow_addons==0.13.0
```

**Comment by Steeeephen** [comment]
Hey @dkbarn 

This is probably caused by you using a more recent python version than we used while training (the conda env file pins the python version to avoid this). It shouldn't be a problem if you use a more recent TF version, especially if you're just using it for inference

Give it a try and if it breaks we can open a new issue

**Comment by JemiloII** [comment]
I don't use conda though and won't use it. I don't want different versions of python floating around for a single project. If it can't use the same python version for all my packages, it's not worth for me.

**Comment by Steeeephen** [comment]
Yeah that's no problem at all! If you want to stick to a particular Python version then that shouldn't be a issue. At the end of the day the version we pinned is just what we used while training the model, the conda env was added just for ease of use and reliability

The important part of this project is the SavedModel (not the code, which is just here for transparency and for anyone who wants to further finetune the model) which should work across any version of TF (or at least 2.0.0+), so install whichever version works for you and everything should still work

The image preprocessing function for inference may break in newer versions - we haven't tested it - but it's basic enough that I doubt it'll break beyond a simple fix. We haven't seen an issue with it yet so we'll cross that bridge when (if) we come to it
